backend/spec/controllers/spree/admin/resource_controller_spec.rb

```diff
@@ -12,7 +12,7 @@ def model_class
   end
 end
 
-describe Spree::Admin::WidgetsController, :type => :controller, :truncation => true do
+describe Spree::Admin::WidgetsController, :type => :controller do
```

Comment:
Not if we have `infer_spec_type_from_file_location` on (which we do), but they seem to be used and there's no harm (and maybe we'll turn that off eventually).